SRCH-3046 Update README with standardized search-services language (#942)
GSA/search-gov: README.md

The required services (MySQL, Elasticsearch, etc.) can be run using Docker. Please refer to [searchgov-services](https://github.com/GSA/search-services) for detailed instructions on centralized configuration for services.

The Elasticsearch service provided by `searchgov-services` is configured to run on the default port, [9200](http://localhost:9200/). To use a different host (with or without port) or set of hosts, set the `ES_HOSTS` environment variable. For example, use following command to run the specs using Elasticsearch running on `localhost:9207`:

    ES_HOSTS=localhost:9207 bin/rspec spec

> Verify that Elasticsearch 6.8.x is running on port the expected port (in this example, port 9200):

Reviewer: ```suggestion
Verify that Elasticsearch 6.8.x is running on the expected port (port 9200 by default):
```
